src/Sentry/SentryLaravel/SentryLaravelServiceProvider.php

```diff
@@ -105,18 +85,16 @@ public function register()
             ), $user_config));
 
             // In Laravel <5.3 we can get the user context from here
-            if (version_compare($app::VERSION, '5.3') < 0) {
-                if (isset($user_config['user_context']) && $user_config['user_context'] !== false) {
+            if (isset($user_config['user_context']) && $user_config['user_context'] !== false && version_compare($app::VERSION, '5.3') < 0) {
+                try {
                     // Bind user context if available
-                    try {
-                        if ($app['auth']->check()) {
-                            $client->user_context(array(
-                                'id' => $app['auth']->user()->getAuthIdentifier(),
-                            ));
-                        }
-                    } catch (\Exception $e) {
-                        error_log(sprintf('sentry.breadcrumbs error=%s', $e->getMessage()));
+                    if ($app['auth']->check()) {
+                        $client->user_context(array(
+                            'id' => $app['auth']->user()->getAuthIdentifier(),
+                        ));
                     }
+                } catch (Exception $e) {
+                    error_log(sprintf('sentry.breadcrumbs error=%s', $e->getMessage()));
```

Comment:
Any chance this should be named `sentry.user_context` maybe, instead of …breadcrumbs?